tests/cloud_functions/token_refresh/test_main.py

```
@@ -0,0 +1,117 @@
+
+
+@patch(
+    "cloud_functions.token_refresh.main.partial_log_structured",
+    new=lambda *a, **kw: None,
+)
+@patch(
+    "cloud_functions.token_refresh.main.strava_refresh_oauth_token",
+    new=lambda *a, **kw: None,
+)
+@patch("firebase_admin.auth.verify_id_token")
+def test_token_refresh_missing_data_source(
```

Ruff (0.8.2)

Missing return type annotation for public function `test_token_refresh_missing_data_source`

Add return type annotation: `None`

(ANN201)